examples/spiders/manual_recaptcha.py

```diff
@@ -60,6 +59,5 @@ def make_screenshot(self, response: PuppeteerScreenshotResponse, **kwargs):
         with open("imageToSave.png", "wb") as fh:
             fh.write(base64.b64decode(data))
 
-    @staticmethod
-    def error(failure: Failure):
-        print(f"We are in error function!")
+    def error(self, failure: Failure):
+        self.log("We are in error function!")
```

Comment:
Do you want to log this as simple message, or like error with error level and stacktrace or any meta info?

Reply:
This is just a simple message that something is bad with the spider. Nothing more.

Reply:
Sounds like warning